### PR TITLE
enh(UI): Allow detail tiles to be re-ordered

### PR DESCRIPTION
**Pull request by Thebarda**
## Description

This adds the capacity to re-order the details tiles.
https://user-images.githubusercontent.com/12515407/126010522-715997ef-0990-48eb-a827-0782d262e1ce.mov

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 20.04.x
- [ ] 20.10.x
- [x] 21.04.x
- [x] 21.10.x (master)

<h2> How this pull request can be tested ? </h2>

- Open the details for a service
- Move the tiles with your mouse like a drag and drop
- -> The tiles are correctly re-ordered
- Close the panel
- Open the details for another service
- -> The details tiles are ordered in the same way like the other service 

## Checklist

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by Jordan91 — DISMISSED** [review]
That working but it's little bit it is difficult for me to understand all the changes ^^

File: www/front_src/src/Resources/Details/tabs/Details/index.tsx [hunk 1]
```diff
@@ -1,55 +1,22 @@
...
-import DetailsCard from './DetailsCard';
-import getDetailCardLines from './DetailsCard/cards';
-import CommandWithArguments from './CommandLine';
+import SortableCards from './SortableCards';
 
 const useStyles = makeStyles((theme) => ({
-  details: {
-    display: 'grid',
-    gridRowGap: theme.spacing(2),
+  fab: {
```
Comment (bdauria) [inline www/front_src/src/Resources/Details/tabs/Details/index.tsx:14]: Is this used?

**Comment by bdauria** [comment]
I will test this out as soon as the build passes.

**Comment by bdauria** [comment]
@Thebarda My remarks so far:

- It seems to me that the shadow is too pronounced, getting a bit too adressive to the eye in my opinion. I would keep the same level as the one we had before (we probably have a double one because of the added container now, just get rid of the extra one).
- It would look better in my opinion if we'd get rid of the rouned corners in the inside container:
Capture d’écran 2021-08-03 à 09 00 43
- There seem to have a reordering issue when the panel is resized up and then down:
tiles

**Review by bdauria — DISMISSED** [review]
LGTM & tested

**Comment by bdauria** [comment]
@Thebarda Little confilct here :)

**Comment by Thebarda** [comment]
> @Thebarda Little confilct here :)

I'v forgot to push the conflict resolution

**Review by bdauria — DISMISSED** [review]

**Review by jdelpierre — DISMISSED** [review]

**Comment by jdelpierre** [comment]
I tested it, and I think it's really great! Great job @Thebarda 💯 👍

**Review by jdelpierre — DISMISSED** [review]

**Review by jdelpierre — DISMISSED** [review]

**Review by jdelpierre — APPROVED** [review]

**Review by bdauria — APPROVED** [review]

**Pull request merged**